src/registrar/tests/test_views_domain.py

```diff
@@ -759,7 +759,7 @@ def test_domain_invitation_flow(self):
         new_user.on_each_login()
 
         # Now load the home page and make sure our domain appears there
-        home_page = self.app.get(reverse("home"))
+        home_page = self.app.get(reverse("user-profile"))
         self.assertContains(home_page, self.domain.name)
```

Comment:
When a new user first logs in, they are taken to the `user-profile` page, instead of the `home`.

Reply:
@zandercymatics if I revert this test, and added the 
`self.assertNotContains(home_page, email_address)`, it would fail since its technically on the home page. Is that what you meant?